src/hooks/engine.rs

```diff
@@ -962,6 +990,23 @@ pub struct dma_t {
     pub buffer: *mut c_uchar,
 }
 
+#[repr(C)]
+pub struct cl_entity_s {
+    pub index: c_int,
+    pub padding1: [c_char; 4 + 340 * 3],
+    pub current_position: c_int,
+    pub padding2: [c_char; 1792 + 8 + 52 + 4],
+    pub origin: [c_float; 3],
+    pub angles: [c_float; 3],
+}
+
+#[repr(C)]
+pub struct client_state_t {
+    pub max_edicts: c_int,
+    pub padding1: [c_char; 1717500 - 4],
```

Comment:
Same comment about offset being different, only in this case you did add separate `cl_viewent`?